ci/wheel_smoke_test_cudf.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import cudf
+import pyarrow as pa
+
+if __name__ == '__main__':
+    n_legs = pa.array([2, 4, 5, 100])
+    animals = pa.array(["Flamingo", "Horse", "Brittle stars", "Centipede"])
+    names = ["n_legs", "animals"]
+
+    print(animals)
+
+    foo = pa.table([n_legs, animals], names=names)
+    print(foo)
+
+    df = cudf.DataFrame.from_arrow(foo)
+    print(df)
+    print(df.loc[df["animals"] == "Centipede"]["n_legs"])
```

Comment:
I'd drop the prints, but maybe worth adding an extra assert or two (e.g. this final equality check could be an assertion.